packages/fuel-gauge/src/bytecode-sway-lib.test.ts

```diff
@@ -1,25 +1,26 @@
-import { FUEL_NETWORK_URL, Predicate, Provider, arrayify } from 'fuels';
+import { Predicate, arrayify } from 'fuels';
+import { launchTestNode } from 'fuels/test-utils';
 
-import { FuelGaugeProjectsEnum, getFuelGaugeForcProject } from '../test/fixtures';
 import { defaultPredicateAbi } from '../test/fixtures/abi/predicate';
 import { defaultPredicateBytecode } from '../test/fixtures/bytecode/predicate';
+import { BytecodeSwayLibAbi__factory } from '../test/typegen/contracts';
+import BytecodeSwayLibAbiHex from '../test/typegen/contracts/BytecodeSwayLibAbi.hex';
 
-import { getSetupContract } from './utils';
+import { launchTestContract } from './utils';
 
 /**
  * @group node
+ * @group browser
  */
 describe('bytecode computations', () => {
-  test('compute_bytecode_root', async () => {
-    const { binHexlified: bytecodeFromFile } = getFuelGaugeForcProject(
-      FuelGaugeProjectsEnum.CALL_TEST_CONTRACT
-    );
-
-    const setupContract = getSetupContract(FuelGaugeProjectsEnum.BYTECODE_SWAY_LIB);
-    const contract = await setupContract();
+  it('compute_bytecode_root', async () => {
+    using contract = await launchTestContract({
+      deployer: BytecodeSwayLibAbi__factory,
+      bytecode: BytecodeSwayLibAbiHex,
+    });
 
     const { waitForResult } = await contract.functions
-      .compute_bytecode_root(arrayify(bytecodeFromFile))
+      .compute_bytecode_root(Array.from(arrayify(BytecodeSwayLibAbiHex)))
```

Comment:
`compute_bytecode_root` expects a `Vec<BigNumberish>` but `arrayify` returns a `Uint8Array` hence the transformation here, @danielbate any ideas?

Reply:
@maschad Was this working before?

Reply:
Yes it, but the test setup is substantially different, the [`getSetupContract` returns a more generic `Contract` type ](https://github.com/FuelLabs/fuels-ts/blob/master/packages/fuel-gauge/src/bytecode-sway-lib.test.ts#L14-L22)which has a ` InvokeFunctions[string](...args: any[]):` as opposed to the types which are narrowed here as a result of typegen.  If you look at this file locally in your editor locally you can see the type inferential differences between this branch and master.

Reply:
This should be fixed in #2784.